tests/calico_testcases/example_policies/testcase15-ports/testcase15-with-ingress-scheme.yaml

```diff
@@ -0,0 +1,118 @@
+namespaceList: ../../example_podlist/ns_list.json
+podList: ../../example_podlist/service_and_ingress_resources
+
+networkConfigList:
+  - name: ip
+    networkPolicyList:
+      - ingress_list.yaml
+    expectedWarnings: 0
+
+  - name: global-simple
+    networkPolicyList:
+      - testcase15-global-simple.yaml
+    expectedWarnings: 0
+
+  - name: global-simple-with-ingress
+    networkPolicyList:
+      - ingress_list.yaml
+      - testcase15-global-simple.yaml
+    expectedWarnings: 0
+
+  - name: global-not-simple
+    networkPolicyList:
+      - testcase15-global-not-simple.yaml
+    expectedWarnings: 0
+
+  - name: global-not-simple-with-ingress
+    networkPolicyList:
+      - ingress_list.yaml
+      - testcase15-global-not-simple.yaml
+    expectedWarnings: 0
+
+  - name: named-ports
+    networkPolicyList:
+      - testcase15-named-ports.yaml
+    expectedWarnings: 12
+
+  - name: equiv-games1
+    networkPolicyList:
+      - testcase15-equiv-games1.yaml
+    expectedWarnings: 0
+
+  - name: equiv-games2
+    networkPolicyList:
+      - testcase15-equiv-games2.yaml
+    expectedWarnings: 0
+    
+  - name: equiv-games3
+    networkPolicyList:
+      - testcase15-equiv-games3.yaml
+    expectedWarnings: 0
+
+  - name: equiv-games4
+    networkPolicyList:
+      - testcase15-equiv-games4.yaml
+    expectedWarnings: 0
+    
+  - name: named-ports-rectangles
+    networkPolicyList:
+      - testcase15-named-ports-rectangles.yaml
+    expectedWarnings: 0
+
+queries:
+
+  - name: connectivity_map
```

Comment:
Missing comparison to expected output on this query.

Reply:
Added